README.md

```diff
@@ -1,6 +1,6 @@
 # React Clock
-- Replace `<your_account>` with your Github username in the
- [DEMO LINK](https://<your_account>.github.io/react_clock/)
+- Replace `SeverusOwl` with your Github username in the
```

Comment:
In this task you need to set a random name only with random number (no strings after 'Set random name' just random number)
1. Add `Set random name` button to the `App` to set a random number as a `clockName` variable in the `App` state.